spec/store/index-spec.js

```diff
@@ -307,13 +307,15 @@ describe("Store", () => {
       expect(store.notebook).toEqual(commutable.toJS(nb));
     });
 
-    xit("should return a fully-fledged notebook when the file isn't empty", () => {
```

Comment:
Previously the test had used `xit()`, so I think the test was never run, especially because it would have failed.

Reply:
This was marked as expect fail intentionally because the error didn't make sense and we didn't want to hold up ci.

Reply:
Ohh that makes sense. Is the updated test ok now?